.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Description
```

Comment:
This file is a separate thing, right? Maybe you want to land it in a separate PR.